src/components/dashboard/feedstock-card.js

```diff
@@ -0,0 +1,77 @@
+import { useRepo } from '@/lib/endpoints'
+import { TimeDeltaFormatter } from '@/lib/time-delta'
+import {
+  Box,
+  Heading,
+  LinkBox,
+  LinkOverlay,
+  Stack,
+  Text,
+  VStack,
+  Skeleton,
+} from '@chakra-ui/react'
+import React from 'react'
+import { GoMarkGithub } from 'react-icons/go'
+import { Link } from '@/components'
+
+export const FeedstockCard = ({ spec, id }) => {
+  const href = `/dashboard/feedstock/${id}`
+
+  const { repo, repoError } = useRepo(spec)
+
+  if (repoError) return <Box>{'error'}</Box>
+  if (!repo) return <Skeleton minH={'100vh'}></Skeleton>
+
+  return (
+    <LinkBox
+      p={4}
+      rounded='lg'
+      transitionProperty='all'
+      transitionDuration='slower'
+      transitionTimingFunction='ease-out'
+      bg='gray.50'
+      _dark={{ bg: 'gray.700' }}
+      _hover={{
+        transform: 'scale(1.025)',
+        boxShadow: 'var(--chakra-shadows-md)',
+      }}
+    >
+      <VStack spacing={2} align='stretch'>
+        <LinkOverlay
+          href={href}
+          as={Link}
+          _hover={{
+            textDecoration: 'none',
+          }}
+        >
+          <VStack spacing={2} align='stretch'>
+            <Heading as={'h3'} size='xs' textTransform='uppercase'>
+              {spec
+                .toLowerCase()
+                .replace('pangeo-forge/', '')
+                .replace('-feedstock', '')}
+            </Heading>
+            <Text lineHeight='tall' opacity={0.8}>
+              {repo.commit.message}
```

Comment:
I'm getting a clientside error on the index page -- looks like it's related to the `recipe.commit` being undefined?

<img width="887" alt="Screen Shot 2022-06-21 at 11 16 17 AM" src="https://user-images.githubusercontent.com/12436887/174870159-6f36e084-40a9-4176-b0f0-825bb8303157.png">

Reply:
The culprit here is GitHub's aggressive secondary rate limiting (due to unauthenticated requests).  i'm going to address this as part of 

- https://github.com/pangeo-forge/pangeo-forge.org/issues/105